src/Nelmio/Alice/Fixtures/Builder/Methods/ReferenceRangeName.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Nelmio\Alice\Fixtures\Builder\Methods;
+
+use Nelmio\Alice\Fixtures\Fixture;
+use Nelmio\Alice\Instances\Collection;
+
+final class ReferenceRangeName implements MethodInterface
+{
+    /**
+     * @var array $matches
+     */
+    private $matches = [];
+
+    /**
+     * @var \Nelmio\Alice\Instances\Collection
+     */
+    private $objects;
+
+    /**
+     * ReferenceRangeName constructor.
```

Comment:
I would remove the PHPDoc here, he doesn't tell anything about the function that's not already included in the method signature.